tests/conftest.py

```diff
@@ -1,7 +1,26 @@
-import lldb
 import pytest
 
+from hilda.exceptions import LLDBException
 from hilda.hilda_client import HildaClient
+from hilda.launch_lldb import LLDBAttachName
+from hilda.lldb_importer import lldb
+
+PROCESS = 'sysmond'
+
+
+@pytest.fixture(scope='session')
+def lldb_debugger(request):
+    lldb_t = None
+    try:
+        lldb_t = LLDBAttachName(PROCESS)
+        lldb_t.start()
+        yield lldb_t.debugger
+    except LLDBException as e:
+        pytest.exit(f'{e.message} - Try use sudo')
```

Comment:
let's catch a more specific exception for that

Reply:
discussed